### DISCUSSION TITLE
Can I set the epoch/step initial value to 1?

### DISCUSSION THREAD

**Discussion by yuys0602** [Lightning-AI/pytorch-lightning#8054] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
Can I set the epoch/step initial value to 1? Now the initial default is 0, it feels awkward when watching Tensorboard.
In addition, can I bring plots in the training and valid in one tersorboard plot?

**Comment by tchaton** [comment, marked as answer]
Dear @yuys0602,

We are starting at 0 as we are counting the number of epoch and before starting, you actually haven't completed any epoch.
If this is really an issue for you, I believe there might be a hacky solution.

I believe this should be working if you are using master. @carmocca Could you confirm ?

```
def training_step()

    self.log("epoch", self.trainer.current_epoch + 1)
```

Concerning, `can I bring plots in the training and valid in one tersorboard plot?` 
Unfortunately, I believe this is a feature provided or not by Logger and Lightning can't do much about it.

Best,
T.C

**Reply by yuys0602** [reply]
thanks a lot
